apps/api/src/app/workflows-v1/workflow-v1.controller.ts

```diff
@@ -227,7 +227,7 @@ export class WorkflowControllerV1 {
         data: body.data,
         __source: query?.__source,
         type: WorkflowTypeEnum.REGULAR,
-        origin: WorkflowOriginEnum.NOVU_CLOUD,
+        origin: WorkflowOriginEnum.NOVU_CLOUD_V1,
```

Comment:
Do we need a data migration in Mongo for existing workflows?

Reply:
We don't need but we can, there is still way to distinguish without migrating